clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1485,6 +1485,43 @@ default DescribeFeaturesResult describeFeatures() {
      */
     UpdateFeaturesResult updateFeatures(Map<String, FeatureUpdate> featureUpdates, UpdateFeaturesOptions options);
 
+    /**
+     * Unregisters a broker.
```

Comment:
What does this mean exactly? The verb "decommission" sounds like we are permanently removing the broker and reassigning any partitions that were assigned to it.

Reply:
Right. I guess at some point the comment and the name of the API seem to have diverged. Updated the comment to reflect the use of the API